FLT/HaarMeasure/ModularCharacter.lean

```diff
@@ -4,6 +4,23 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Andrew Yang, Yaël Dillies, Javier López-Contreras
 -/
 import Mathlib.MeasureTheory.Measure.Haar.Unique
+import FLT.Mathlib.Data.ENNReal.Inv
+import FLT.ForMathlib.DomMulActMeasure
+
+/-!
+# The modular character of Haar measures
+
+Given a group `G` acting on an measurable additive commutative group `A`, and an element `g : G`,
+one can pull back the Haar measure `μ` of `A` along the map `(g • ·) : A → A` to get another Haar
+measure `μ'` on `A`. By unicity of Haar measures, there exists some nonnegative real number `r` such
+that `μ' = r • μ`. We can thus define a map `modularHaarChar : G → ℝ≥0` sending `g` to its
+associated real number `r`. Furthermore, this number doesn't depend on the Haar measure `μ` we
+started with, and `modularHaarChar` is a group homomorphism.
```

Comment:
In the literature "modular character" means something else unfortunately, so I don't think we should call it this. In the literature, `A = Additive G` and the action is just translation, which is not an action of G on the group A; it is a left action on the set A which commutes with the right action of A on itself by addition. I think we need to stick to the literature when it comes to naming. I don't know what the name of what you're defining is but you are assuming `DistribMulAction` so I don't think it can be "modular character". See for example https://math.stackexchange.com/questions/2237765/how-is-the-modular-character-usually-defined for the sign confusion in the literature, but you're doing something else as far as I can see.